src/components/web-form-renderer.vue

```diff
@@ -193,7 +193,8 @@ const fetchSubmissionAttachments = () => {
   const requestUrl = apiPaths.submissionAttachments(form.projectId, form.xmlFormId, props.instanceId);
   return submissionAttachments.request({
     url: requestUrl,
-    alert: false
+    alert: false,
+    responseType: 'blob', // Handle all file types for attachments.
```

Comment:
Do we need `responseType: 'blob'` here in `fetchSubmissionAttachments()`, or just in `fetchSubmissionAttachment()` below? The response here is an array, and I'm pretty sure it's just metadata. I don't think it contains any binary attachment content.